src/entities/DAO.ts

```diff
@@ -3,21 +3,25 @@ import { LogManager } from 'aurelia-framework';
 import { includeEventsIn, Subscription } from 'aurelia-event-aggregator';
 import { SchemeInfo } from "../entities/SchemeInfo";
 import { Web3Service, BigNumber } from "../services/Web3Service";
-
+import { GlobalConstraintInfo } from "../entities/GlobalConstraintInfo";
 export class DAO extends Organization {
 
   public address: string;
   public name: string;
   private schemesCache: Map<string, SchemeInfo>;
+  private constraintsCache: Map<string, GlobalConstraintInfo>;
   private registerSchemeEvent;
   private unRegisterSchemeEvent;
+  private addConstraintEvent;
+  private removeConstraintEvent;
   public arcService: ArcService;
-  private logger = LogManager.getLogger("Vanille");
+  private logger = LogManager.getLogger("Alchemy");
```

Comment:
Why is this Alchemy?

Reply:
Probably a merge conflict error. I changed it back to Vanille